addons/web/static/src/scss/base_frontend.scss

```diff
@@ -1,5 +1,8 @@
 // Frontend general
 #wrapwrap {
+    // The z-index is useful to prevent that children with a negative z-index
```

Comment:
Two FIXME for this commit:

FIXME: does not work well on default pages because the content has no
background color -> should propagate the bg color on the right element
somehow.

FIXME: when the page content is not long enough, the bottom half of
the footer is shown. In that case, the effect should simply be disabled
so that the footer scrolls normally (like Odoo.com).